src/cmd/build.js

```diff
@@ -45,11 +45,20 @@ export type LocalizedNameParams = {
   manifestData: ExtensionManifest,
 }
 
+// This defines the _locales/messages.json type. See:
+// https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales
+type LocalizedMessageData = {
+  [messageName: string]: {
+    description: string,
+    message: string,
+  },
+}
```

Comment:
yeah, it's unfortunate that flow syntax needs its own linter rules. We should go through all available rules and set them up they way we want it.